lib/matplotlib/tests/test_colorbar.py

```diff
@@ -308,3 +310,34 @@ def test_colorbar_lognorm_extension():
     cb = ColorbarBase(ax, norm=LogNorm(vmin=0.1, vmax=1000.0),
                       orientation='vertical', extend='both')
     assert cb._values[0] >= 0.0
+
+
+@image_comparison(baseline_images=['extended_cbar_with_contourf_min',
```

Comment:
Please condense the test down to a single image with 3 subplots, saving time and space.